src/main/io/ledstrip.c

```diff
@@ -924,15 +939,19 @@ void updateLedStrip(void)
 
     uint32_t now = micros();
 
-    bool indicatorFlashNow = (int32_t)(now - nextIndicatorFlashAt) >= 0L;
-    bool warningFlashNow = (int32_t)(now - nextWarningFlashAt) >= 0L;
-    bool rotationUpdateNow = (int32_t)(now - nextRotationUpdateAt) >= 0L;
+    bool indicatorFlashNow = (now >= nextIndicatorFlashAt);
```

Comment:
`(int32_t)(now - nextIndicatorFlashAt) >= 0L` will work always (value is correct when micros wrap),  `(now >= nextIndicatorFlashAt);` will not work when now is  between `(~0 - nextIndicatorFlashAt) .. ~0`